frontend/src/context/SystemContext.tsx

```diff
@@ -109,11 +109,11 @@ export function SystemContextProvider(
         filter((connected) => !!connected),
         switchMap(() =>
           combineLatest([
-            merge(
-              rpc.chain.subscribeNewHeads(),
-              interval(2000).pipe(switchMap(() => rpc.chain.getHeader()))
-            ).pipe(distinctCodecChanged()),
-            interval(2000).pipe(
+            interval(6000).pipe(
+              switchMap(() => rpc.chain.getHeader()),
+              distinctCodecChanged()
+            ),
+            interval(6000).pipe(
               switchMap(() => rpc.system.health()),
               distinctCodecChanged()
             ),
```

Comment:
This looks dangerous to me, I don't really like intervals in JS because they tend to go in the wrong direction. Would be nicer to have some kind of reactive behavior with the rate limiting for how often we request data and such.

Reply:
Noted, moved to new Issue #73